Readme.md

```diff
@@ -370,6 +370,24 @@ If this option is not specified, the default device configured for the app in Wa
 Type: `string` <br/>
 Default: `undefined`
 
+### Language
+
+To set the language for a session you can use the `appium:language` capability.
+This key allows you to specify the language in which the app will run during the session.
+
+```ts
+{
+  "capabilities": {
```

Comment:
Capabilities are never a key in a dictionary with the object directly under in wdio conf, they are always an array:

```ts
capabilities: [
  { "appium:language": "fr" },
]
```